### PR TITLE
feat(icons): add icons for kayaking, transportation and tour 

### PR DESCRIPTION
**Pull request by Anthropic**
## 🖼️ add icons for kayaking, transportation and tour 

The following icons are required by Luxe if it is to use roo-ui icons

* add icon for kayaking from Material Design update
* add icon for transportation from Material Design update
* add icon for tour from Material Design update

This PR also moves some bespoke images from maps into qantas category

## Looks like this

image
image
image

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [ ] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Angus](https://github.com/angusfretwell), [Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), or [Long](https://github.com/KieraDOG)).
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [ ] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -71,6 +71,12 @@ $ yarn run generate-component MyNewComponent
 Right now we don't have any automation setup for optimising assets.
 Run svgs through a tool like [SVGO](https://jakearchibald.github.io/svgomg/). It will remove attributes and comments from the file which are not needed.
 
+### Icons
+
+Bespoke icons should be added to `src/icons/qantas`.
+
+Icons taken from [Material Design](https://material.io/resources/icons/?style=baseline) should be added to the same category (`src/icons/<category>`) as they appear there.
```
Comment (memcc) [inline .github/CONTRIBUTING.md:78]: nice! 👍

File: src/icons/index.d.ts [hunk 2]
```diff
@@ -925,9 +926,11 @@ declare module "roo-ui/icons" {
   export const star: Icon;
   export const cake: Icon;
   export const domain: Icon;
+  export const emojiTransportation: Icon;
```
Comment (memcc) [inline src/icons/index.d.ts:929]: suggestion: what do you think about just calling this `transportation`?

Comment (Anthropic) [reply src/icons/index.d.ts:929]: Yeah the name is stupid, but it is what MD are using for some reason, I've been doing a lot of ctrl+F to find icons so I am not sure if we should diverge from their naming even though I hate it. We also risk them re-using the name we choose if we go with something different.

Perhaps we can bring it up at tomorrow's front end working group. I can change later with another PR if we wish to re-name then as by tomorrow still no one would be using it. Setting up aliases is another option for their newer poorly named icons.

**Review by memcc — APPROVED** [review]
🚀

File: src/icons/index.d.ts (hunk 2 above)
Comment (memcc) [reply src/icons/index.d.ts:929]: Cool, as discussed we can release as-is, and will chat about it at the next FE working group.

**Review by philipwindeyer — APPROVED** [review]
🚚

**Pull request merged**